src/MoBi.Assets/MoBi.Assets.csproj

```diff
@@ -26,6 +26,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="OSPSuite.Assets" Version="10.0.159-unroltob" />
+    <PackageReference Include="OSPSuite.Assets.Images" Version="10.0.159-unroltob" />
+    <PackageReference Include="OSPSuite.Core" Version="10.0.159-unroltob" />
```

Comment:
This is still outdated here. 

Something is off with the nugget packages

Reply:
I have updated to the latest nuget version and fixed the duplication.